karate-core/README.md

```diff
@@ -75,6 +75,7 @@
     | <a href="#minimize"><code>minimize()</code></a>
     | <a href="#fullscreen"><code>fullscreen()</code></a>
     | <a href="#quit"><code>quit()</code></a>
+    | <a href="#emulatedevice"><code>driver.emulateDevice</code></a>
```

Comment:
move this (and the para) to the "Chrome section" 
<img width="979" alt="image" src="https://user-images.githubusercontent.com/915480/87900349-c4cb6280-ca71-11ea-89d6-6b405643bcac.png">

Reply:
Ok.